app/controllers/api/v1/admin/users_controller.rb

```diff
@@ -25,6 +25,16 @@ def create
           end
         end
 
+        def destroy
+          # TODO: Will need to add additional logic later
+          user = User.find(params[:id])
+          if user.destroy
+            render_data
+          else
+            render_error errors: user.errors.to_a
```

Comment:
**IMHO**, for `render_data` it's common that a success will have a `:ok` status.
So using it in such cases without explicitly define the status code is natural.
But, for `render_error` there's no common error status code each error is specific and self descriptive.
So, I think  for errors even if the  method default status is `:bad_request` it may be better to explicitly express it when calling it -> `render_error errors: user.errors.to_a, status: :bad_request`.
Having the default status value for `render_error` may be better to be omitted.

Reply:
I agree - but I find this is something we should do to the whole application sometime in the future to make sure its all consistent.